lib/plug/static.ex

```diff
@@ -185,7 +185,7 @@ defmodule Plug.Static do
       segments = Enum.map(segments, &uri_decode/1)
 
       if invalid_path?(segments) do
-        raise InvalidPathError
+        raise InvalidPathError, message: "invalid path for static asset #{conn.request_path}"
```

Comment:
Thanks for the suggestion!
Now I notice that both styles are used (see https://github.com/fchabouis/plug/blob/e3c42647a2935416e1d809c0b0513bdbb9e741c8/lib/plug/router.ex#L454 for example)

In case anybody reads this, I've found in the [documentation](https://hexdocs.pm/elixir/1.12/Kernel.html#defexception/1) the part explaining this behavior, that you can either pass to `raise` a string, that will be considered the exception message, or a set of keywords that will be merged into the exception struct.

Reply:
The code that uses the keyword approach is likely old code. :)